crates/sovereign-sdk/module-system/sov-modules-api/src/lib.rs

```diff
@@ -13,6 +13,156 @@ mod reexport_macros;
 #[cfg(feature = "macros")]
 pub use reexport_macros::*;
 
+#[macro_export]
+macro_rules! native_debug {
+    (?$($k:ident).+) => (
```

Comment:
have you considered putting these under sov-modules-macros?

Reply:
I did but macro crate can only export procedural macros, not declarative macros